modules/inflowwind/CMakeLists.txt

```diff
@@ -51,17 +51,8 @@ set(IFW_SOURCES
 add_library(ifwlib ${IFW_SOURCES})
 target_link_libraries(ifwlib nwtclibs)
 
-# LINUX
-add_library(ifw_c_lib driver/IFW_C.f90)
-
-# MAC
-# add_library(ifw_c_lib SHARED driver/IFW_C.f90 ${IFW_SOURCES})
-# set_target_properties(ifw_c_lib PROPERTIES PREFIX "" SUFFIX ".dylib")
-
-# WINDOWS
-# add_library(ifw_c_lib SHARED driver/IFW_C.f90 ${IFW_SOURCES})
-# set_target_properties(ifw_c_lib PROPERTIES PREFIX "" SUFFIX ".dll")
-
+# c-bindings interface library
+add_library(ifw_c_lib SHARED src/IfW_C.f90)
```

Comment:
@andrew-platt what do you think about adding a CMake flag to compile all C-bound interfaces instead of compiling them by default?

Reply:
I like that idea.  We'll need to tie those optional targets to the respective regression tests as we add them.